book/website/communication/open/embedding-ds-ai-disciplines.md

```diff
@@ -0,0 +1,142 @@
+
+**Effective feedback in AfL**: Effective feedback requires active listening from both the educator and the student. As part of the questioning process, it is an essential tool for developing students' thinking. Feedback must be task-focused, timely, specific, clear and unbiased. In this way, you will provide your students with information about their current performance and guidance on how they can improve to reach their goals.
+
+2. **Formative Assessment (FA)** supports teaching by assessing a learner's state and inferring next steps (Zhai et al., 2021). It is similar to AfL, as both methods are used to understand student progress and inform teaching. However, while AfL is carried out during learning to inform teaching and identify areas for improvement, FA is used for day-to-day assessments to gauge and explore students' understanding of a topic.
+
+The FA process usually consists of the following three practices (Stanja et al., 2023):
+* **Eliciting**: The collection of evidence for students' learning using tasks and questions 
+* **Interpreting**: Analysing what students are saying, writing or doing and what this indicates about their thinking; and identifying implications for learning based on the previous analysis
+* **Responding**: Giving feedback to students or adaptation of instruction. 
+
+Formative assessments in data science and AI education should focus on providing timely and actionable feedback that helps students improve their understanding and skills progressively. Examples include:
+* **Code reviews**: Regularly assessing students’ code for functionality, efficiency, and style.
+* **Concept checks**: Quick, informal assessments during lessons, such as quizzes or hands-on tasks, to gauge understanding of recent topics like machine learning algorithms or statistical methods.
+* **Data challenges**: Mini group competitions where students predict outcomes based on given datasets, which are then discussed in class to learn from various approaches. When teaching in multidisciplinary settings, the challenges should be situated in the disciplinary domain that is most familiar to students.
+
+3. **Differentiated Instruction (DI)** — a method that considers students’ individual learning styles and levels of readiness before designing a lesson plan (Tomlinson, 2017). 
+
+DI sits between "single-size" instruction and individualised instruction, involving proactive planning of various ways for students to express their learning. While it may require fine-tuning for individual learners, offering multiple options increases the likelihood of effectively meeting the needs of many students.
```

Comment:
```suggestion
Differentiated instruction sits between "single-size" instruction and individualised instruction, involving proactive planning of various ways for students to express their learning. 
While it may require fine-tuning for individual learners, offering multiple options increases the likelihood of effectively meeting the needs of many students.
```